.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!--
+⚡️ katchow! We ❤️ Pull Requests! If you remove or skip this template, you'll
+make the 🐼 sad and the mighty god of GitHub will appear and pile-drive the
+close button from a great height while making animal noises.
+
```

Comment:
```suggestion
```

Sorry, but this is not the tone this org would use

Reply:
Our social spaces are to remain _dignified_ environments and this is way out of line